tests/src/test/java/alluxio/shell/AbstractAlluxioShellTest.java

```diff
@@ -90,6 +95,12 @@ protected void copyToLocalWithBytes(int bytes) throws IOException {
     fileReadTest("/testFile", 10);
   }
 
+  /**
+   * Reads the local file copied from Alluxio and tests whether its size is equal to {@code size}.
```

Comment:
this doesn't just check the size, it checks all the data as well with `BufferUtils.equalIncreasingByteArray`